chroma-agent/Target

```diff
@@ -53,44 +53,57 @@ The name of the target.
 END
 }
 
+dbg() {
+    echo "`date` RA $1" >> /var/log/chroma-agent.log
```

Comment:
I understand, was commenting on two processes writing to one log file being sub-optimal.